src/components/swap/SwapModalFooter.tsx

```diff
@@ -120,46 +154,136 @@ export default function SwapModalFooter({
+      value: ReactNode | string
+      labelTooltipText?: string
+      color?: string
+    }> = [
+      { label: t`Exchange rate`, value: `${'1 ' + labelInverted + ' = ' + formattedPrice ?? '-'} ${label}` },
+      {
+        label: t`Network fee`,
+        value: `~${formatCurrencyAmount(trade.gasUseEstimateUSD, NumberType.FiatGasPrice)}`,
+        labelTooltipText: t`The fee paid to miners who process your transaction. This must be paid in ${nativeCurrency.symbol}.`,
+      },
+      {
+        label: t`Price impact`,
+        value: trade.priceImpact ? formatPriceImpact(trade.priceImpact) : '-',
+        color: getPriceImpactWarning(trade.priceImpact),
+        labelTooltipText: t`The impact your trade has on the market price of this pool.`,
+      },
+    ]
+    if (trade.tradeType === TradeType.EXACT_INPUT) {
+      details.push({
+        label: t`Minimum received`,
+        value: `${trade.minimumAmountOut(allowedSlippage).toSignificant(6)} ${trade.outputAmount.currency.symbol}`,
+        labelTooltipText: t`The minimum amount you are guaranteed to receive. If the price slips any further, your transaction will revert.`,
+      })
+    } else {
+      details.push({
+        label: t`Maximum sent`,
+        value: `${trade.maximumAmountIn(allowedSlippage).toSignificant(6)} ${trade.inputAmount.currency.symbol}`,
+        labelTooltipText: t`The maximum amount you are guaranteed to spend. If the price slips any further, your transaction will revert.`,
+      })
+    }
+
+    return details
+  }, [allowedSlippage, nativeCurrency.symbol, trade])
 
   return (
     <>
-      <AutoRow>
-        <TraceEvent
-          events={[BrowserEvent.onClick]}
-          element={InterfaceElementName.CONFIRM_SWAP_BUTTON}
-          name={SwapEventName.SWAP_SUBMITTED_BUTTON_CLICKED}
-          properties={formatAnalyticsEventProperties({
-            trade,
-            hash,
-            allowedSlippage,
-            transactionDeadlineSecondsSinceEpoch,
-            isAutoSlippage,
-            isAutoRouterApi: !clientSideRouter,
-            swapQuoteReceivedDate,
-            routes,
-            fiatValueInput: fiatValueInput.data,
-            fiatValueOutput: fiatValueOutput.data,
-          })}
-        >
-          <ButtonError
-            onClick={onConfirm}
-            disabled={disabledConfirm}
-            style={{ margin: '10px 0 0 0' }}
-            id={InterfaceElementName.CONFIRM_SWAP_BUTTON}
+      <Column gap="md" padding="0 0.5rem">
+        {details.map(({ label, value, color, labelTooltipText }, i) => (
+          <SwapModalDetailRow key={i} label={label} value={value} color={color} labelTooltipText={labelTooltipText} />
+        ))}
+      </Column>
+      {showAcceptChanges ? (
+        <SwapShowAcceptChanges justify="flex-start" gap="0px" data-testid="show-accept-changes">
+          <RowBetween>
+            <RowFixed>
+              <AlertTriangle size={20} style={{ marginRight: '8px', minWidth: 24 }} />
```

Comment:
Shouldn't we wrap this with Styled Components to pass that style? Also, looks like `px` is missing for `minWidth`.